code/Test_definitions/number-recycling.feature

```diff
@@ -1,12 +1,12 @@
-Feature: CAMARA Number Recycling API, 0.1.0-rc.1 - Operation number-recycling
+Feature: CAMARA Number Recycling API, v0.1 - Operation number-recycling
   # Environment variables:
   # * api_root: API root of the server URL
   #
-  # References to OAS spec schemas refer to schemas specifies in number-recycling.yaml, version 0.1.0-rc.1
+  # References to OAS spec schemas refer to schemas specifies in number-recycling.yaml, version 0.1.0
```

Comment:
If you omit the redundant information in this place, the file is easier to maintain in releases:
```suggestion
  # References to OAS spec schemas refer to schemas specifies in number-recycling.yaml
```